package.json

```diff
@@ -84,10 +84,10 @@
     "@storybook/test-runner": "0.7.1",
     "@storybook/theming": "6.5.10",
     "@svgr/webpack": "5.5.0",
-    "@types/enzyme": "3.10.12",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "12.1.5",
     "@testing-library/user-event": "14.4.3",
+    "@types/enzyme": "3.10.12",
```

Comment:
Sorry I've actually got one more request. Could you move this back to where it was above? Otherwise it looks like a change. Also the `package-lock.json` changes should be included in this PR after running an `npm i`.